book/src/keymap.md

```diff
@@ -335,32 +335,22 @@ experience.
 | `Backspace`, `Ctrl-h`                       | Delete previous char        | `delete_char_backward`  |
 | `Delete`, `Ctrl-d`                          | Delete next char            | `delete_char_forward`   |
 
-However, if you really want navigation in insert mode, this is supported. An
-example config that gives the ability to use arrow keys while still in insert
-mode:
-
-```toml
-[keys.insert]
-"up" = "move_line_up"
-"down" = "move_line_down"
-"left" = "move_char_left"
-"right" = "move_char_right"
-"C-b" = "move_char_left"
-"C-f" = "move_char_right"
-"A-b" = "move_prev_word_end"
-"C-left" = "move_prev_word_end"
-"A-f" = "move_next_word_start"
-"C-right" = "move_next_word_start"
-"A-<" = "goto_file_start"
-"A->" = "goto_file_end"
-"pageup" = "page_up"
-"pagedown" = "page_down"
-"home" = "goto_line_start"
-"C-a" = "goto_line_start"
-"end" = "goto_line_end_newline"
-"C-e" = "goto_line_end_newline"
-"A-left" = "goto_line_start"
-```
+### Insert Mode (beginners)
+
+These keys are not recommended, it is for those who do not read documentations.
+
+| Key                                         | Description                 | Command                 |
+| -----                                       | -----------                 | -------                 |
+| `Up`                                        | Move to previous line       | `move_line_up`          |
+| `Down`                                      | Move to next line           | `move_line_down`        |
+| `Left`                                      | Backward a char             | `move_char_left`        |
+| `Right`                                     | Forward a char              | `move_char_right`       |
+| `Ctrl-Left`                                 | Backward a word             | `move_prev_word_end`    |
+| `Ctrl-Right`                                | Forward a word              | `move_next_word_start`  |
+| `PageUp`                                    | Move one page up            | `page_up`               |
+| `PageDown`                                  | Move one page down          | `page_down`             |
```

Comment:
Page down/up will have to be removed too if we're going with only arrow keys and home/end.

Reply:
Should we also support page down/up since it is one special key keyboard have that is similar to arrow keys and home/end, and it works without pressing `C-`? If another I think page up/down is more useful than home/end.